tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -895,6 +896,7 @@ def forward_xp(self, inputs, xp):
     ((1, 1, 2), -4)
 ])
 def test_expand_dims_invalid(xp, shape, axis):
+    warnings.simplefilter('error', DeprecationWarning)
```

Comment:
Could you create a new pull request for this fix?

Reply:
Created: #7824